hudi-hadoop-mr/pom.xml

```diff
@@ -36,6 +36,17 @@
       <artifactId>hudi-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hudi</groupId>
+      <artifactId>hudi-client-common</artifactId>
```

Comment:
`HoodieMemoryConfig`, `HoodieCommonConfig` and `HoodieReaderConfig` containing all "real-time" configs are already in `hudi-common` module on master so there is no need to add `hudi-client-common` module dependency.